bower.json

```diff
@@ -24,10 +24,10 @@
     "preview.png"
   ],
   "dependencies": {
-    "polymer": "polymer/polymer#^1.1.0"
+    "polymer": "polymer/polymer#~1.9"
```

Comment:
Polymer recommends such selector for hybrid components: `"polymer": "Polymer/polymer#1.9 - 2"`

See: https://www.polymer-project.org/2.0/docs/devguide/hybrid-elements